core/server/master/src/main/java/alluxio/master/file/InodePermissionChecker.java

```diff
@@ -33,22 +32,19 @@ static InodePermissionChecker create() {
    *
    * @param user the user
    * @param groups the groups that user belongs to
-   * @param path the path
    * @param inode inode of the path
    * @param permission the permissions to check
-   * @throws AccessControlException when user does not have the permission
+   * @return whether the user has the required permission
    */
-  void checkPermission(String user, List<String> groups, String path, Inode<?> inode,
-      Mode.Bits permission) throws AccessControlException;
+  boolean checkPermission(String user, List<String> groups, Inode<?> inode, Mode.Bits permission);
```

Comment:
Just curious, why return `boolean` now?

Reply:
`PermissionChecker` will throw `AccessControlException`. There is no need for every implementation of `InodePermissionChecker` to throw that exception.